### PR TITLE
Travis build setup to commit generated files to a travis-build branch.

### PR DESCRIPTION
**Pull request by dlabrecq**
Note that the publish.sh script depends on properties (i.e., AUTH_TOKEN and TARGET_BRANCH) being set via Travis CI configuration.

### CODE REVIEW

**Comment by dlabrecq** [comment]
@dgutride @priley86 and/or @dtaylor113 please review.

File: publish.sh [hunk 1]
```diff
@@ -0,0 +1,18 @@
+#/bin/sh
+
+# User info
+git config --global user.name "Admin"
+git config --global user.email "patternfly@redhat.com"
+
+# Add upstream authentication token
+git remote add upstream https://$AUTH_TOKEN@github.com/dlabrecq/patternfly-org.git
```
Comment (dgutride) [inline publish.sh]: Will this change to @github.com/patternfly/patternfly-org.git or do we need a bit more testing?

Comment (dlabrecq) [inline publish.sh]: Ah, yes. I need to change that. Thanks!

**Comment by dgutride** [comment]
Looks good

**Comment by dtaylor113** [comment]
LGTM

**Comment by priley86** [comment]
@dlabrecq  - per our conversation we can also git add `_site` in `publish.sh`. I was testing this out in gh-pages and we will also need this directory. 

Looks good to me otherwise. Good job!

**Comment by dlabrecq** [comment]
FYI, it is not necessary to add the _site directory. This directory built as part of OpenShift hot deploy and output directly to the php server. If we decided to use gh-pages, we can consider including it. For now, I think we should omit it.

**Pull request merged**